src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-101700.mdx

```diff
@@ -0,0 +1,75 @@
+
+## New features
+
+* Adds support for custom tasks in Celery
+
+  * Add monitoring support for custom task classes in [Celery](https://pypi.org/project/celery/).  Previously, only tasks using the default task class were able to be monitored.  Note: This version of the Python agent will no longer support Celery versions less than [v5.2.0](https://github.com/celery/celery/releases/tag/v5.2.0).
+
+* Adds support for Graphene-Django
+
+  * Add monitoring support for synchronous schemas created with [Graphene-Django](https://pypi.org/project/graphene-django/).  Previously, schemas created with Graphene-Django would only be partially monitored, resulting in missing [GraphQL](https://pypi.org/project/graphql-core/) related attributes.
+
+* Adds support for middleware filtering in Django
+
+  * Users are now able to filter which [Django](https://pypi.org/project/django/) middleware to monitor.  There are now [three additional settings](/docs/apm/agents/python-agent/configuration/python-agent-configuration/#middleware-filters):
+  
+    * `instrumentation.middleware.django.enabled` (Default is `true`)
+    * `instrumentation.middleware.django.exclude`
+    * `instrumentation.middleware.django.include`
+  
+  See the [Django Middleware Filtering page](/docs/apm/agents/python-agent/supported-features/django-middleware-filtering/) for more information and examples.
+
+* Adds support for AWS Bedrock Converse API
+
+  * Add monitoring support for non-streaming [Converse](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/bedrock-runtime/client/converse.html) API.
+
+## Bug fixes
+
+* Add safeguard for deepest unique path in GraphQL
+
+  * Some users were experiencing issues with the deepest unique path in [GraphQL](https://pypi.org/project/graphql-core/) not being generated due to a field name returning `None`. This issue has been fixed. 
+
+* Add safeguard for web request parsing
+
+  * Some users experienced a parsing error when `request_uri` was malformed or contained invalid characters.  This can happen at this point if the `request_uri` in the request header is invalid and does not match the (valid) URL originally given.  While this safeguard has now been put in place in New Relic, `urllib` has resolved this issue for versions of Python released after Jan 31, 2025.
+
+* Fix crash in ElasticSearch and GraphQL when `trace.settings==None`
+
+  * In cases where `trace.settings` is `None`, the instrumentation for ElasticSearch crashed the application.  A `global_settings` fallback has been put in place.  This also resolves the same issue found in GraphQL.
+
+* Remove `options` method from async ElasticSearch client list
+
+  * The previous instrumentation of ElasticSearch caused a crash because `options` was in the list of async methods.  This method does not need to be wrapped in a `DatastoreTrace` since it is not one that conducts any datastore operations/requests.
```

Comment:
```suggestion
  * The previous instrumentation of ElasticSearch caused a crash because `options` was in the list of async methods.  This method is no longer wrapped in a `DatastoreTrace` since it is not one that conducts any datastore operations/requests.
```